app/javascript/packs/application.js

```diff
@@ -7,30 +7,35 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-import "jquery"
-import "jquery-ujs"
-import "bootstrap"
-import "bootstrap-toggle"
-import "moment"
-import "d3"
-import "c3"
-import "underscore"
-import {Spinner} from "spin.js"
-import "clipboard"
-import * as Cookies from "js-cookie"
-import "tipsy"
-import "jquery.turbolinks"
+import "jquery";
+import "jquery-ujs";
+import "bootstrap";
+import "bootstrap-toggle";
+import "moment";
+import "d3";
+import "c3";
+import "underscore";
+import {Spinner} from "spin.js";
+import "clipboard";
+import * as Cookies from "js-cookie";
+import "tipsy";
+import "jquery.turbolinks";
 
-global.moment = moment
-global.c3 = c3
-global._ = underscore
-global.Spinner = Spinner
-global.Clipboard = clipboard
-global.Cookies = Cookies
+global.moment = moment;
+global.c3 = c3;
+global._ = underscore;
+global.Spinner = Spinner;
+global.Clipboard = clipboard;
+global.Cookies = Cookies;
 
 $(function() {
-    $('input[type=checkbox][data-toggle^=toggle]').bootstrapToggle()
-})
-
-import "../convert"
+    $('input[type=checkbox][data-toggle^=toggle]').bootstrapToggle();
+});
 
+import "../convert";
+import "../highchart_theme.js";
+import "../graphs/pb_graph.js";
+import "../graphs/segment_history_mean_graph.js";
+import "../graphs/run_duration_graph.js";
+import "../graphs/segment_duration_graph.js";
+import "../graphs/reset_graph.js";
```

Comment:
I forget if we're on ES6, but if we are semicolons are optional and generally not used

Reply:
Anything in `app/javascript` is part of webpack so definitely es6 there, I just wasn't sure what the standard was so I opted to put them just in case.